python/tvm/relay/transform.py

```diff
@@ -400,6 +401,23 @@ def CombineParallelConv2D(min_num_branches=3):
     return _transform.CombineParallelConv2D(min_num_branches)
 
 
+def CombineParallelDense(min_num_branches=3):
+    """Combine multiple dense operators into one.
+
+    Parameters
+    ----------
+    min_num_branches : int
+        The minimum number of required parallel branches for performing this
```

Comment:
I think it is good to add an example of the optimization in the doc string so end-users can understand when it is useful to apply this optimization.